### PR TITLE
Adds some memorable brain damage lines from myself

### PR DESCRIPTION
**Pull request by ghost**
https://tgstation13.org/phpBB/viewtopic.php?f=34&t=12052

### CODE REVIEW

**Comment by kevinz000** [comment]
attention whore

**Comment by XDTM** [comment]
At least brain damage lines are confined to one trauma now

**Comment by PKPenguin321** [comment]
These are gay like u

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -62,7 +62,9 @@
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
-        "DON'T EVER TUCH ME"
+        "DON'T EVER TUCH ME",
+        "@pick(semicolon)i didn't know people would run into the shard!",
+        "@pick(semicolon)i think i got bsa'd cause i read the worst cat porn i could find over comms..."
```
Comment (duncathan) [inline strings/brain_damage_lines.json]: Nobody cares; revert these but keep the Terry addition that's fine

**Review by duncathan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**